src/modules/jcms_rest/src/Plugin/rest/resource/PersonListRestResource.php

```diff
@@ -102,18 +101,10 @@ protected function filterSubjects(QueryInterface &$query) {
   /**
    * Apply filter for type by amending query.
    */
-  protected function filterType(QueryInterface &$query) {
-    $type = $this->getRequestOption('type');
-    if (!is_null($type)) {
-      $entityManager = \Drupal::service('entity_field.manager');
-      $fields = $entityManager->getFieldStorageDefinitions('node', 'person');
-      $options = options_allowed_values($fields['field_person_type']);
-      if (!in_array($type, array_keys($options))) {
-        throw new JCMSBadRequestHttpException(t('Invalid type'));
```

Comment:
We're losing the ability to inform the client that the type is not recognised but this isn't strictly required. We may want to distinguish between a `type` that produces no results and a `type` that isn't recognised. Don't feel strongly about it. This can go.